kolibri/core/exams/serializers.py

```diff
@@ -137,6 +137,12 @@ def to_internal_value(self, data):
             else:
                 # Otherwise we are just updating the exam, so allow a partial update
                 self.partial = True
+
+        question_sources = data.get("question_sources", [])
```

Comment:
Might be good to update this: https://github.com/learningequality/kolibri/blob/develop/kolibri/core/exams/test/test_exam_api.py#L45 to remove this from the data we send to the backend in tests, and then add a test to assert that the value is properly set on create?

Reply:
I can do this in #12200 to keep the clean-up separate from the fix here.